client/src/components/frontend/ProjectTexts.js

```diff
@@ -0,0 +1,191 @@
+import React, { Component, PropTypes } from 'react';
+import { Link } from 'react-router';
+
+export default class ProjectTexts extends Component {
+
+  static propTypes = {};
+
+  render = () => {
```

Comment:
Doesn't necessarily need to happen in this pull request, but even without real data, this component could be broken up into smaller pieces. The GroupedTexts components should take a collection of texts and chunk them by tag. It should loop through each tag chunk, render the tag, then loop through each text and render the text (thumbnail, title, stats, etc). The bit that renders the text should be its own component.